handbook/company/product-groups.md

```diff
@@ -28,8 +28,8 @@ At Fleet, [anyone can contribute](https://fleetdm.com/handbook/company#openness)
 
 | Product group                            | Goal _(value for customers and/or community)_                                                                          | Capacity |
 |:-----------------------------------------|:-----------------------------------------------------------------------------------------------------------------------|:---------|
-| [MDM](#mdm-group)                        | Increase and exceed maturity in the [device management](https://fleetdm.com/device-management) product category.       | 104      |
-| [Orchestration](#orchestration-group)    | Increase and exceed maturity in the [orchestration](https://fleetdm.com/orchestration) product category.               | 104      |
+| [MDM](#mdm-group)                        | Increase and exceed maturity in the [device management](https://fleetdm.com/device-management) product category.       | 117      |
+| [Orchestration](#orchestration-group)    | Increase and exceed maturity in the [orchestration](https://fleetdm.com/orchestration) product category.               | 117      |
 | [Software](#software-group)              | Increase and exceed maturity in the [software management](https://fleetdm.com/software-management) product category.   | 104      |
```

Comment:
```suggestion
| [MDM](#mdm-group)                        | Increase and exceed maturity in the [device management](https://fleetdm.com/device-management) product category.       | 130      |
| [Orchestration](#orchestration-group)    | Increase and exceed maturity in the [orchestration](https://fleetdm.com/orchestration) product category.               | 117      |
| [Software](#software-group)              | Increase and exceed maturity in the [software management](https://fleetdm.com/software-management) product category.   | 104      |
```

Reply:
I think Martin is allocated a full point sprint so MDM would be 5 * 26 = 130. Correct me if I'm wrong @georgekarrv 

I think Juan is allocated a half point sprint so Orchestration would be (4 * 26) + 13 = 117. Does that sound correct @sharon-fdm? 

Software is 4 * 26 = 104.

Reply:
UPDATE: Just merged it in. Please let us know if we're wrong.

Reply:
Sorry for not getting to this yet. I would prefer to keep Jordan on 13sp estimate for 1 more sprint while he still ramps up but if we understand this would kick in starting May 26 I think that would be fair